spec/indent_spec.rb

```diff
@@ -102,6 +102,36 @@ def number
     expect(@indent.indent(input)).to eq output
   end
 
+  it 'should properly indent nested code with consecutive linebreaks' do
```

Comment:
> 1. Avoid `should`, just say `it 'properly indents...'`

👍. This was to be consistent with the surrounding descriptions, but I agree with avoiding `should` in this manner. Changed to reflect this guidance.

> 2. Since input is the same as output, I would make it a variable and reuse in the expectation

It's hard to see on GitHub, but the empty line on [111](https://github.com/pry/pry/blob/b02d9be44dc28771baef10568bb4e881f04677ae/spec/indent_spec.rb#L111) becomes a line with two spaces on [124](https://github.com/pry/pry/blob/b02d9be44dc28771baef10568bb4e881f04677ae/spec/indent_spec.rb#L124). Inserting hash signs (as specified in [the comment at the top of the file](https://github.com/pry/pry/blob/b02d9be44dc28771baef10568bb4e881f04677ae/spec/indent_spec.rb#L3-L5)) affects the test case, since the test case is about empty lines.

Given that context, what would you think about
```ruby
input = [
  'class C',
  '  def foo',
  '    :foo',
  '  end',
  '',
  '  def bar',
  '    :bar',
  '  end',
  'end'
].join("\n")

output = [
  'class C',
  '  def foo',
  '    :foo',
  '  end',
  '  ',
  '  def bar',
  '    :bar',
  '  end',
  'end'
].join("\n")
```
to prevent accidental editor whitespace linting, or something like
```ruby
output = input.lines
output[4] = '  ' + output[4]
output = output.join
```
to express the diff between `input` and `output`? For now, I updated to the second one.

The above snippets are attempts to give you something concrete to comment on or perhaps say "yes" to, but ultimately, my preference here is "whatever @kyrylo & other maintainers prefer". 

https://user-images.githubusercontent.com/261376/125472890-d38a2f85-dd13-443b-8a37-1e8cfb38a729.mp4

> 3. I would change the description to say something like "given already indented code it returns self". That is, Indent didn't make any changes

This makes sense 👍; your example description for the submitted code is clearer than the original. I'm making this change. Let me know if, given your thoughts of the above, I should update the description (while keeping it clear!).